packages/worker/src/workflow/reusable-vm.ts

```diff
@@ -3,6 +3,8 @@ import { AsyncLocalStorage } from 'node:async_hooks';
 import vm from 'node:vm';
 import * as internals from '@temporalio/workflow/lib/worker-interface';
 import { IllegalStateError } from '@temporalio/common';
+import { getTimeOfDay } from '@temporalio/core-bridge';
```

Comment:
I know this is not new, but any reason why getting nano time through core was preferred over [process.hrtime](https://nodejs.org/api/process.html#processhrtimebigint)?

Reply:
We need to compare timestamps of logs emitted by Core and Lang when buffering Core logs so they can be emitted in order.

Reply:
From the hrtime docs:
> These times are relative to an arbitrary time in the past, and not related to the time of day and therefore not subject to clock drift. The primary use is for measuring performance between intervals:

We need time of day in nanosecond resolution.